src/frontend/web/src/ratingpage/Ratingpage.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
```

Comment:
Please change the name of the file to `RatingPage` - to use the standard JS conventions.